UE/Application/Application.cpp

```diff
@@ -105,4 +105,9 @@ void Application::handleCallDrop(){
     context.state->handleCallDrop();
 }
 
+void Application::handleSendCallDrop(common::PhoneNumber sender, common::PhoneNumber receiver)
+{
+    context.state->handleSendCallDrop(sender,receiver);
```

Comment:
Hmm w sumie już była na to metoda, ale ok ta jest bardziej ogólna, bo tamta działała dla accept i detach to można było by z niej tu skorzystać w tym tasków do dropów, ale może być  będzie więcej metod do wyboru :). Tamta to handleSendCallRequest(phoneNumber, bool) i boolean = true Accept boolean = false Drop.